src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -25,12 +39,12 @@ protected PythonNodeBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel>
             ArgumentLacing = LacingStrategy.Disabled;
         }
 
-        public static readonly string DefaultPythonEngine = "IronPython2";
-        public static readonly string PythonNet3Engine = "CPython3";
+        internal PythonEngineVersions DefaultPythonEngine = PythonEngineVersions.IronPython2;
 
-        private string engine = DefaultPythonEngine;
+        private PythonEngineVersions engine = PythonEngineVersions.IronPython2;
 
-        public string Engine
+        [JsonIgnore]
+        public PythonEngineVersions EngineEnum
```

Comment:
For enums that are not flags (i.e. can be combined using `|`) the name should be singular. In this case it would be `PythonEngineVersion`.